app/styles/app/modules/switch.sass

```diff
@@ -43,7 +45,9 @@
 
   .on
     display: none
-    margin-left: 0
+    margin-left: -49%
```

Comment:
hmm sadly removing that rule causes the switches to look wrong on the hooks page:

![image](https://user-images.githubusercontent.com/43280/29424460-746804f4-8345-11e7-93ea-15cdaed977cc.png)

But I’ll find a workaround, the C of CSS is a perpetual problem 😆